applications/nrf_desktop/README.rst

```diff
@@ -62,143 +62,187 @@ The project contains configuration files for following boards:
+A connected BLE peer device can be controlled using predefined buttons or button combinations.
+There are several peer operations available.
 
-    For the gaming mouse the dedicated hardware switch is used to select the dongle peer.
-    When the dongle peer is selected, the peer control is disabled until the switch will be set to another position.
+The peer operation states provide visual feedback through LEDs (if the device has LEDs).
+Each of the states is represented by separate LED color and effect.
 
-* **bond erase**
+Gaming mouse
+++++++++++++
+
+The following predefined buttons or button combinations are assigned to peer control operations for the gaming mouse:
+
+Hardware switch:
+    * The switch is located next to the optical sensor.
+    * You can set the switch in the following positions:
+
+        * Top position: Connection to the dongle peer active.
+        * Middle position: Connection to other BLE peers active.
+        * Bottom position: Mouse turned off.
+        
+      When the dongle peer is selected, the peer control is disabled until the switch is set to another position.
+
+Precision Aim button:
+    * The button is located on the left side of the mouse, in the thumb area.
+    * Short click to initialize the peer selection.
+      During the peer selection:
+        
+        1. Short click to select the next peer.
+        #. Double click to confirm the peer selection. 
+           The peer is changed after the confirmation.
+
+    * Long click to initialize the peer erase, then double click to confirm the operation.
+      After the confirmation, Bluetooth advertising using a new local identity is started.
+      When a new Bluetooth central successfully connects and establishes security, the old bond is removed and the new bond is used instead.
+      If the new peer does not connect in the predefined period of time, the advertising ends and the application switches back to the old peer.
+    * Press the button for time longer than short click, but shorter than long click to cancel ongoing peer operation.
+
+Desktop mouse
++++++++++++++
+
+The following predefined buttons are assigned to peer control operations for the desktop mouse:
+
+Scroll wheel button:
+    * The scroll wheel must be pressed before the mouse is powered up.
+        
+        * After the confirmation, Bluetooth advertising using a new local identity is started.
+          When a new Bluetooth central successfully connects and establishes security, the old bond is removed and the new bond is used instead.
+          If the new peer does not connect in the predefined period of time, the advertising ends and the application switches back to the old peer.
+    
+    * Press the button for time longer than short click, but shorter than long click to cancel ongoing peer operation.
+
+Keyboard
+++++++++
+
+The following predefined buttons or button combinations are assigned to peer control operations for the keyboard:
+
+Page Down key:
```

Comment:
Maybe: Page Down with Fn key modifier?

Reply:
You press only one repeatedly, so I would keep this as it is.

Reply:
ok